### PR TITLE
Move central store thread to utils

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

File: tendrl/performance_monitoring/utils/central_store_util.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+    try:
+        summary = ClusterSummary(
+            cluster_id=cluster_id
+        )
+        if not summary.exists():
+            raise TendrlPerformanceMonitoringException(
+                "No summary found for cluster %s" % cluster_id
+            )
+        summary = summary.load().to_json()
+        if '_etcd_cls' in summary:
```
Comment (r0h4n) [inline tendrl/performance_monitoring/utils/central_store_util.py]: you dont need all these checks, use below checks

https://github.com/Tendrl/commons/blob/develop/tendrl/commons/objects/__init__.py#L305

File: tendrl/performance_monitoring/utils/central_store_util.py [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        summary = SystemSummary(
+            sds_type=cluster_type
+        )
+        if not summary.exists():
+            raise TendrlPerformanceMonitoringException(
+                "No clusters of type %s found" % cluster_type
+            )
+        summary = summary.load().to_json()
+        if '_etcd_cls' in summary:
+                del summary['_etcd_cls']
```
Comment (r0h4n) [inline tendrl/performance_monitoring/utils/central_store_util.py]: same as above

File: tendrl/performance_monitoring/utils/central_store_util.py [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+    exs = ''
+    if node_ids is None:
+        node_ids = get_node_ids()
+    for node_id in node_ids:
+        try:
+            current_node_summary = etcd_read(
+                '/monitoring/summary/nodes/%s' % node_id
+            )
+            if '_etcd_cls' in current_node_summary:
+                del current_node_summary['_etcd_cls']
```
Comment (r0h4n) [inline tendrl/performance_monitoring/utils/central_store_util.py]: same as above

File: tendrl/performance_monitoring/utils/central_store_util.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/utils/central_store_util.py]: Done

File: tendrl/performance_monitoring/utils/central_store_util.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/utils/central_store_util.py]: Done

File: tendrl/performance_monitoring/utils/central_store_util.py (hunk 3 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/utils/central_store_util.py]: Done

**Comment by anmolbabu** [comment]
Need to verify

**Comment by anmolbabu** [comment]
Verified

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

**Pull request merged**